pkg/updater/resultstore/resultstore.go

```diff
@@ -39,6 +40,25 @@ import (
 // check if interface is implemented correctly
 var _ updater.TargetResult = &singleActionResult{}
 
+type TestResultStatus int64
+
+const (
+	// TestResultUnspecified is v2.TestCase.RESULT_UNSPECIFIED.
+	TestResultUnspecified = 0
+	// TestCompleted is v2.TestCase.COMPLETED.
+	TestCompleted = 1
+	// TestInterrupted is v2.TestCase.INTERRUPTED.
+	TestInterrupted = 2
+	// TestCancelled is v2.TestCase.CANCELLED.
+	TestCancelled = 3
+	// TestFiltered is v2.TestCase.FILTERED.
+	TestFiltered = 4
+	// TestSkipped is v2.TestCase.SKIPPED.
+	TestSkipped = 5
+	// TestSuppressed is v2.TestCase.SUPPRESSED.
+	TestSuppressed = 6
+)
+
```

Comment:
again - we don't need this intermediary translation layer, please convert it directly in your `mapStatusToCellResult` function below